.github/workflows/web_build.yml

```
@@ -0,0 +1,105 @@
+        uses: docker/setup-qemu-action@v3
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+      - name: Login to DockerHub
+        uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - name: Get options
+        id: options
+        env:
+          TAG: ${{ needs.info.outputs.tag_short }}
```

actionlint

property "tag_short" is not defined in object type {name: string; new_tag: string; new_tag_short: string; sha_short: string}

(expression)